src/cmd/completion.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var completionNoDesc bool
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+Bash:
+
+$ source <(git-town completion bash)
+
+# To load completions for each session, execute once:
+Linux:
+  $ git-town completion bash > /etc/bash_completion.d/git-town
+MacOS:
+  $ git-town completion bash > /usr/local/etc/bash_completion.d/git-town
```

Comment:
Global?

Reply:
Do you mean install globally for all users? I think most people would prefer to do this locally for each user.

Reply:
Right, for dotfile repos.